diff --git a/node_modules/@mobily/ts-belt/dist/types/global.d.ts b/node_modules/@mobily/ts-belt/dist/types/global.d.ts
index 38ae2c1..515c4a3 100644
--- a/node_modules/@mobily/ts-belt/dist/types/global.d.ts
+++ b/node_modules/@mobily/ts-belt/dist/types/global.d.ts
@@ -1,7 +1,7 @@
-export {}
+export {};
 
 declare global {
-  namespace Belt {
-    type UseMutableArrays = 0
-  }
+	namespace Belt {
+		type UseMutableArrays = 1;
+	}
 }
